compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -361,6 +361,42 @@ flatbuffers::Offset<circle::Buffer> encodeOpBufferByDType(FlatBufferBuilder &bui
   return CreateBuffer(builder, array_offset);
 }
 
+template <>
+flatbuffers::Offset<circle::Buffer>
+encodeOpBufferByDType<loco::DataType::S4>(FlatBufferBuilder &builder, luci::CircleConst *c)
+{
+  const uint32_t size = c->size<loco::DataType::S4>();
+  const uint32_t raw_size = (size + 1) / 2;
+  std::vector<uint8_t> raw_data(raw_size);
+
+  printf("=== circle data\r\n");
+  for (uint32_t i = 0; i < size; ++i)
+  {
+    printf("%2d: %d\r\n", i, int32_t(c->at<loco::DataType::S4>(i)));
```

Comment:
I guess that value of `c` must be within [-8, 7].

Reply:
yup